lib/css/components/_stacks-usercards.less

```
@@ -0,0 +1,89 @@
+//
+//  STACK OVERFLOW
+//  USER CARDS
+//
+//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
+//  Stack Overflow. For documentation of all these classes and how to contribute,
+//  visit https://stackoverflow.design/
+//
+//  ============================================================================
+//  $  BASE
+//  ----------------------------------------------------------------------------
+
+
+.s-usercard {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-column-gap: @su8;
+}
+
+.s-usercard--time {
+    grid-column: ~"1 / 3";
+    grid-row: ~"1 / 2";
+    color: var(--black-350);
+    font-size: @fs-caption;
+}
+
+.s-usercard--avatar {
+    grid-column: ~"1 / 2";
+    grid-row: ~"2 / 4";
+}
+
+.s-usercard--name {
+    margin-bottom: 0;
+    grid-column: ~"2 / 3";
+    grid-row: ~"2 / 3";
+    font-size: @fs-caption;
+    font-weight: 500;
+}
+
+.s-usercard--awards {
+    grid-column: ~"2 / 3";
+    grid-row: ~"3 / 4";
+    display: flex;
+
+    li {
+        margin-right: @su6;
+        font-size: @fs-caption;
+    }
+}
+
+.s-usercard--rep {
+    font-weight: 700;
+    padding-top: 2px;
+}
+
+.s-usercard--highlighted {
```

```suggestion
.s-usercard__highlighted {
```

This is a modifier and should be underscores.